allennlp/modules/seq2seq_encoders/intra_sentence_attention.py

```diff
@@ -43,13 +44,16 @@ class IntraSentenceAttentionEncoder(Seq2SeqEncoder):
         :func:`~allennlp.nn.util.combine_tensors`; see that function for more detail on exactly how
         this works, but some simple examples are ``"1,2"`` for concatenation (the default),
         ``"1+2"`` for adding the two, or ``"2"`` for only keeping the attention representation.
+    output_projection_dim : ``bool``, optional (default = None)
```

Comment:
Maybe just call this `output_dim`, with the docstring saying that if it's given, we'll do a projection to this dimension?